include/s3select_functions.h

```diff
@@ -15,6 +15,11 @@ using namespace std::string_literals;
 namespace s3selectEngine
 {
 
+constexpr double sec_scale(int n)
+{
+    return pow(10, n);
+}
+
```

Comment:
#if BOOST_DATE_TIME_POSIX_TIME_STD_CONFIG
      const double scale = sec_scale(9-1);    //nano-sec
    #else
      const double scale = sec_scale(6-1);    //micro-sec
    #endif
    
    no need to repeat 9 times this part, 
    /const double scale/ could be defined globally(and static)